src/EncompassRest/Loans/RateLocks/CommitmentTypes.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.RateLocks
+{
+    /// <summary>
+    /// CommitmentTypes
+    /// </summary>
+    public enum CommitmentTypes
```

Comment:
Could you rename to `CommitmentType`. Plural enum names are usually reserved for Flag enums.

Reply:
Resolved in 721cae4

Reply:
I just noticed there was already a CommitmentType enum created. I switch this to reuse the existing enum instead of creating a new one in 8178100